terraform/modules/alertmanager/templates/alertmanager.tpl

```diff
@@ -67,6 +67,10 @@ route:
     - match_re:
         namespace: verify-doc-checking-prod|verify-dcs-integration
       receiver: dcs-slack
+    - match:
+        namespace: verify-doc-checking-prod
+        severity: p2
+      receiver: "dcs-p2"
```

Comment:
Currently this will NOT be caught as the matcher above it will catch all the things that match "namespace = verify-doc-checking-prod" and send them to slack.

I think what you want is to add some sub routing _below_ that matcher, that further filters on the prod namespace and the p2 severity label.

Like:

```suggestion
      routes:
      - match:
          namespace: verify-doc-checking-prod
          severity: p2
        receiver: "dcs-p2"
```